cluster/test-deploy/gcp-dev/vars-origin.yaml

```diff
@@ -0,0 +1,124 @@
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  },
+  {
+    "name": "oso-rhui-rhel-server-extras",
+    "baseurl": "https://mirror.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://use-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://use-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://euw-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://gce-mirror1.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/
+        https://gce-mirror2.ops.rhcloud.com/libra/rhui-rhel-server-7-extras/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  },
+  {
+    "name": "dockertested",
+    "baseurl": "https://mirror.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://euw-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 1,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  }
+]
+
+openshift_gcp_prefix: "ci-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}-"
+
+openshift_gcp_region: us-east1
+openshift_gcp_zone: us-east1-c
+
+openshift_gcp_network_name: "{{ openshift_gcp_prefix }}network"
+
+openshift_gcp_iam_service_account_keyfile: "{{ files_dir }}/gce.json"
+
+openshift_gcp_ssh_private_key: "{{ files_dir + '/ssh-privatekey' }}"
+
+openshift_gcp_infra_network_instance_group: ig-m
+
+# Instructs the launch job to build an AMI prior to cluster provisioning and then use that image
+openshift_gcp_build_image: true
+openshift_gcp_image: '{{ openshift_gcp_prefix }}images'
+openshift_gcp_base_image: 'rhel-7'
+openshift_gcp_root_image: 'centos-7'
+
+openshift_gcp_registry_bucket_keyfile: "{{ files_dir }}/gce.json"
+openshift_gcp_registry_bucket_name: "origin-ci-test-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}"
```

Comment:
Should this use `openshift_gcp_prefix` instead of `origin-ci-test-`?

Reply:
looks like we probably want another PR that cleans up a lot of the "ci" references... seems aesthetic for now